### PR TITLE
feat: e2e add experiment row "kill" action tests

### PR DESCRIPTION
**Pull request by thiagodallacqua-hpe**
## Ticket

[ET-662](https://hpe-aiatscale.atlassian.net/browse/ET-662)

## Description

This Pr intends to add tests for the "kill" experiment row action for the DataGrid.

## Test Plan

- run the `'Row Actions'` test from the `src/e2e/tests/experimentList.spec.ts` test file.
- all checks must pass

Screenshot 2024-10-30 at 16 56 04

## Checklist

- [ ] Changes have been manually QA'd
- [ ] New features have been approved by the corresponding PM
- [ ] User-facing API changes have the "User-facing API Change" label
- [ ] Release notes have been added as a separate file under `docs/release-notes/`
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses have been included for new code which was copied and/or modified from any external code

[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[ET-662]: https://hpe-aiatscale.atlassian.net/browse/ET-662?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webui/react/src/e2e/tests/experimentList.spec.ts [hunk 1]
```diff
@@ -677,5 +680,47 @@ test.describe('Experiment List', () => {
...
+      const experimentActionDropdown = await newExperimentRow.experimentActionDropdown.open();
+
+      await experimentActionDropdown.menuItem('Kill').pwLocator.click();
+
+      await experimentActionDropdown.stopKillModal.pwLocator.waitFor({ state: 'visible' });
+      await experimentActionDropdown.stopKillModal.footer.pwLocator.getByText('Kill').click();
+      await experimentActionDropdown.stopKillModal.pwLocator.waitFor({ state: 'hidden' });
+
+      // eslint-disable-next-line no-console
+      console.log((await newExperimentRow.getCellByColumnName('State')).pwLocator.innerText());
```
Comment (JComins000) [inline webui/react/src/e2e/tests/experimentList.spec.ts:717]: do you still need this? i have another `browserLog` method if you need

**Review by JComins000 — APPROVED** [review]

**Comment by JComins000** [comment]
just a few small comments. merge when you're ready

File: webui/react/src/e2e/tests/experimentList.spec.ts [hunk 2]
```diff
@@ -677,5 +680,47 @@ test.describe('Experiment List', () => {
...
+        await projectDetailsPage.f_experimentList.dataGrid.getRowByColumnValue(
+          'ID',
+          killExperiment.toString(),
+        );
+
+      const experimentActionDropdown = await newExperimentRow.experimentActionDropdown.open();
+
+      await experimentActionDropdown.menuItem('Kill').pwLocator.click();
+
+      await experimentActionDropdown.stopKillModal.pwLocator.waitFor({ state: 'visible' });
```
[the remark this answers is not in the document]
Comment (JComins000) [reply webui/react/src/e2e/tests/experimentList.spec.ts:712]: 👍

**Review by keita-determined — COMMENTED** [review]
looks like e2e is failing

File: webui/react/src/e2e/tests/experimentList.spec.ts [hunk 3]
```diff
@@ -677,5 +680,45 @@ test.describe('Experiment List', () => {
...
+        response: { project },
+      },
+      newWorkspace: {
+        response: { workspace },
+      },
+    }) => {
+      killExperiment = Number(
+        detExecSync(
+          `experiment create ${fullPath('examples/tutorials/core_api/2_checkpoints.yaml')} --project_id ${project.id}`,
+        ).split(' ')[2],
```
Comment (ashtonG) [inline webui/react/src/e2e/tests/experimentList.spec.ts:695]: i'm noticing that the CI resources are higher with this PR than without. could you try starting this experiment paused?
```suggestion
        detExecSync(
          `experiment create ${fullPath('examples/tutorials/core_api/2_checkpoints.yaml')} --project_id ${project.id} --paused`,
        ).split(' ')[2],
```

**Pull request closed without merging**